tools/runqslower.py

```diff
@@ -132,7 +132,7 @@
 
     // fetch timestamp and calculate delta
     tsp = start.lookup(&pid);
-    if (tsp == 0) {
+    if ((tsp == 0) || (*tsp == 0)) {
```

Comment:
Why `*tsp == 0` check here and below ?

Reply:
Hi @chenhengqi 

Since this patch creates bpf map with pid_max entries from runtime information, I replace the original BPF_HASH with BPF_ARRAY for preventing hash collision and less array map lookup overhead as above.

All array elements are pre-allocated and zero initialized at init time, so we not only need to check `tsp == 0`, but also need to check it contains a valid timestamp. Does it make sense? Thank you.

Reply:
Sounds reasonable. Thanks.